common/test/acceptance/tests/discussion/test_discussion.py

```diff
@@ -425,6 +432,26 @@ def test_discussion_blackout_period(self):
         self.assertFalse(self.thread_page.is_comment_deletable("comment1"))
         self.assertFalse(self.thread_page.is_comment_deletable("comment2"))
 
+    def test_dual_discussion_module(self):
+        """
+        Scenario: There should be two inline discussion module in one courseware unit
+        Now we have two new post button each of discussion module. When we click on
```

Comment:
Use the BDD syntax
Given that ...
When we...
Then ..
And ..
etc.

Look at examples in other tests, such as 

https://github.com/edx/edx-platform/blob/master/common/test/acceptance/tests/lms/test_lms_instructor_dashboard.py#L52